webpack/react_app/redux/actions/job_invocations/index.js

```diff
@@ -0,0 +1,60 @@
+import { isNil } from 'lodash';
```

Comment:
this is no longer used even in core, see https://github.com/theforeman/foreman/commit/dfbb65cf13f06408770205dd91acbe274d8f68e4#diff-2abb6d1defb26fc866851a507eef315c

Reply:
We are trying to avoid using `lodash` (it's heavy)

In your case, i think you can just do:
```jsx
const jobInvocationsInterval = process.env.JOB_INVOCATIONS_POLLING || defaultJobInvocationsPollingInterval;  
```